paywall/src/unlock.js/startup.ts

```diff
@@ -76,6 +76,6 @@ export default function startup(
   // go!
   mainWindow.init()
   wallet.init()
-  checkoutIframeHandler.init()
+  checkoutIframeHandler.init(wallet.useUserAccounts)
```

Comment:
One thing that I want to clarify: is it true _if_ the account is actually provided by us, or is it true if the use of unlock accounts is authorized?

Reply:
In `Wallet.ts`, the decision goes as follows:

```typescript
this.useUserAccounts =
      !this.hasWallet &&
      (config.unlockUserAccounts === true ||
        config.unlockUserAccounts === 'true')
```

So if there is no browser wallet (such as metamask) _and_ the config allows for user accounts, it's true. For any other combination it is false.